src/routes/jira/server/jira-ghe-server-apps.ts

```diff
@@ -0,0 +1,37 @@
+import { NextFunction, Request, Response } from "express";
+import { GitHubServerApp } from "models/github-server-app";
+
+export const JiraGheServerApps = async (
+	req: Request,
+	res: Response,
+	next: NextFunction
+): Promise<void> => {
+	try {
+		req.log.debug("Received Jira GHE server apps page request");
+
+		const { id: installationId } = res.locals.installation;
+		const githubServerBaseUrl = req.query.serverUrl as string;
```

Comment:
/nit
not sure how url is passed in connect app, will it do things like `encodeURIComponent`? But should be minor, should works on most browser.